### ISSUE TITLE
Update of CEF chrome 

### ISSUE THREAD

**Issue by wouterDevelop** [CasparCG/server#1498]
### Description
is it possible to update the CEF lib to te latest version that is used in OBS? 
### Solution suggestion

https://github.com/obsproject/obs-studio/pull/6867/files 

---

**Comment by Julusian** [comment]
The current obs build just barely works for us (see https://github.com/CasparCG/server/issues/1441 for more details on the issues). I did try using their newer builds some months back, but it does not notify us of each frame being produced, so we dont get anything useful from CEF.

OBS has a very different pipeline architecture, and the changes they have done to CEF are unfortonately not compatible with our architecture. Maybe it could work if the way we import the frames is reworked to handle it, or if their cef work is modified to restore the behaviour we need, I dont think I have looked into this yet.

Or alternatively, we should reconsider whether the shared texture flow provides enough benefit to be worth the constant cost it has to maintenance.
